### PR TITLE
[IMP] pre-commit-vauxoo: Add params, help, default and environment variable matches

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: src/pre_commit_vauxoo/cli.py [hunk 1]
```diff
@@ -13,11 +13,95 @@
...
-    pre_commit_vauxoo.main()
+@click.argument("PATHS", type=click.Path(exists=True), nargs=-1)
+@click.option(
+    "--overwrite",
+    "-w",
+    envvar="PRECOMMIT_OVERWRITE_CONFIG_FILES",
+    default=True,
+    show_default=True,
+    help="Overwrite configuration files. "
+    "If True then overwrite the current configuration files. "
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:45]:
```suggestion
    "If True, existing configuration files into the project will be overwritten. "
```

File: src/pre_commit_vauxoo/cli.py [hunk 2]
```diff
@@ -13,11 +13,95 @@
...
+    "If False then re-use the current ones if exists. "
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:46]:
```suggestion
    "If False, then current files will be used, if they exist. "
```

File: src/pre_commit_vauxoo/cli.py [hunk 3]
```diff
@@ -13,11 +13,95 @@
...
+    "Use environment variable PRECOMMIT_OVERWRITE_CONFIG_FILES separated by commas",
+)
+@click.option(
+    "--exclude-autofix",
+    '-x',
+    type=click.Path(exists=True),
+    multiple=True,
+    help="Exclude paths to run the autofix pre-commit configuration"
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:54]:
```suggestion
    help="Exclude paths on which to run the autofix pre-commit configuration"
```

File: src/pre_commit_vauxoo/cli.py [hunk 4]
```diff
@@ -13,11 +13,95 @@
...
+    default=True,
+    show_default=True,
+    help="Overwrite configuration files. "
+    "If True then overwrite the current configuration files. "
+    "If False then re-use the current ones if exists. "
+    "Use environment variable PRECOMMIT_OVERWRITE_CONFIG_FILES separated by commas",
+)
+@click.option(
+    "--exclude-autofix",
+    '-x',
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:51]: Almost all other strings use double quotes

File: src/pre_commit_vauxoo/cli.py [hunk 5]
```diff
@@ -13,11 +13,95 @@
...
+    type=click.Path(exists=True),
+    multiple=True,
+    help="Exclude paths to run the autofix pre-commit configuration"
+    "Use environment variable EXCLUDE_AUTOFIX separated by commas",
+)
+@click.option(
+    "--exclude-lint",
+    '-l',
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:59]: Same here

File: src/pre_commit_vauxoo/cli.py [hunk 6]
```diff
@@ -13,11 +13,95 @@
...
+    type=click.Path(exists=True),
+    multiple=True,
+    help="Exclude paths to check lints. Use environment variable EXCLUDE_LINT separated by commas.",
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:62]: Paths to exclude checks

File: src/pre_commit_vauxoo/cli.py [hunk 7]
```diff
@@ -13,11 +13,95 @@
...
+)
+@click.option(
+    "--disable-pylint-checks",
+    '-d',
+    type=str,
+    envvar="DISABLE_PYLINT_CHECKS",
+    metavar='<columns>',
+    help="Disable pylint checks separated by commas. Use environment variable DISABLE_PYLINT_CHECKS",
```
Comment (luisg123v) [inline src/pre_commit_vauxoo/cli.py:70]:
```suggestion
    help="Pylint checks to disable, separated by commas. Use environment variable DISABLE_PYLINT_CHECKS",
```

**Review by luisg123v — CHANGES_REQUESTED** [review]
Only typos or similar in messages, code LGTM

File: src/pre_commit_vauxoo/cli.py (hunk 4 above)
Comment (moylop260) [reply src/pre_commit_vauxoo/cli.py:52]:
```suggestion
    "-x",
```

File: src/pre_commit_vauxoo/cli.py (hunk 5 above)
Comment (moylop260) [reply src/pre_commit_vauxoo/cli.py:60]:
```suggestion
    "-l",
```

File: src/pre_commit_vauxoo/cli.py (hunk 6 above)
Comment (moylop260) [reply src/pre_commit_vauxoo/cli.py:63]:
```suggestion
    help="Paths to exclude checks. Use environment variable EXCLUDE_LINT separated by commas.",
```

**Review by moylop260 — COMMENTED** [review]
(using gh ui to commit diff)

**Comment by moylop260** [comment]
@luisg123v 

Thanks for review

Could you check it again, please?

File: README.rst [hunk 8]
```diff
@@ -65,6 +65,46 @@ Using github directly
...
+Full --help command result:
+
+Usage: pre-commit-vauxoo [OPTIONS] [PATHS]...
+
+  PATHS are the specific filenames to run hooks on. Also, it can be defined
+  using environment variable INCLUDE_LINT separated by commas [default:
+  Current directory]
+
+Options:
+  -w, --overwrite BOOLEAN         Overwrite configuration files. If True then
```
Comment (luisg123v) [inline README.rst:86]: This help seems outdated

File: src/pre_commit_vauxoo/cli.py [hunk 9]
```diff
@@ -13,11 +13,97 @@
...
 
 
+def strcsv2tuple(strcsv, is_path=False):
+    if not strcsv:
+        return ()
+    items = []
+    for item in strcsv.strip().split(','):
+        item = item.strip()
+        if is_path and not os.path.exists(item):
+            print("getcwd %s and item %s" % (os.getcwd(), item))
```
Comment (moylop260) [inline src/pre_commit_vauxoo/cli.py:30]:
```suggestion

```

**Pull request merged**